### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by apprehensions**
+ Limited `.gitignore` to just `vinegar`
+ Changed desktop files to their shortened names and moved them to `desktop/`
  + Removed comments
+ Removed Logo
+ README
  + Moved TODO
  + Removed license from README
  + Removed 'Thanks'
  + Added 'Features' section
+ Moved `src/` to `/`
+ Moved various functions to `util.go`
+ Made various routines and clean functions for ease of maintenance
  + `DeleteDir(dir string)`
  + `DirsCheck(dir ...string)`
  + `InitExec(dirs *Dirs, path string, url string, what string) (string)`
  + `RbxFpsUnlocker(dirs *Dirs)`
  + `PfxKill(dirs *Dirs)`
+ Added time to logging
+ Added `exec` mainly for `vinegar exec explorer`
+ Fixed safety issues
+ Fixed repeating of code
+ Custom rbxfpsunlocker file

### CODE REVIEW

**Comment by apprehensions** [comment]
To make vinegar a bit faster, it would require fetching and finding the latest versions directory of Roblox Studio and the Roblox Player, which will be difficult in Go.

File: go.mod [hunk 1]
```diff
@@ -1,38 +1,31 @@
-module github.com/nocrusts/vinegar
```
Comment (Thelolguy1) [inline go.mod:1]: need to rebuild this file after merging, outdated dependencies.

File: go.sum [hunk 2]
```diff
@@ -9,20 +9,15 @@ cloud.google.com/go/bigquery v1.0.1/go.mod h1:i/xbL2UlR5RvWAURpBYZTtm/cXjCha9lbf
 cloud.google.com/go/datastore v1.0.0/go.mod h1:LXYbyblFSglQ5pkeyhO+Qmw7ukd3C+pD7TKLgZqpHYE=
```
Comment (Thelolguy1) [inline go.sum:9]: ditto

**Review by Thelolguy1 — APPROVED** [review]
nice work

**Pull request merged**